angular/src/app/app.component.html

```diff
@@ -0,0 +1,60 @@
+<dx-scheduler
+  #targetScheduler
+  timeZone="America/Los_Angeles"
+  [dataSource]="dataSource"
+  [views]="['month']"
+  currentView="month"
+  [currentDate]="currentDate"
+  [firstDayOfWeek]="1"
+  [startDayHour]="8"
+  [endDayHour]="18"
+  [showAllDayPanel]="false"
+  [height]="600"
+  [groups]="['employeeID']"
+  resourceCellTemplate="resourceCellTemplate"
+  dataCellTemplate="dataCellTemplate"
+  appointmentTooltipTemplate="appointmentTooltipTemplate"
+>
+  <dxi-resource
+    fieldExpr="employeeID"
+    [allowMultiple]="false"
+    [dataSource]="resourcesDataSource"
+    label="Employee"
+  >
+  </dxi-resource>
+
+  <ng-container *dxTemplate="let data of 'appointmentTooltipTemplate'">
+
+    <!-- NOTE: You can use data.appointmentData.resouceId to obtain resource color -->                                  
+    <Tooltip
+        [appointmentData]="data.appointmentData"
+        [isDeleteButtonExist]="isDeleteButtonExist(data)"
+        [markerColor]="getColor | apply : data.appointmentData.employeeID"
+        (onDeleteButtonClick)="onDeleteButtonClick($event, data.appointmentData)"
+    ></Tooltip>
+  </ng-container>
+
+  <div *dxTemplate="let employee of 'resourceCellTemplate'">
+    <div class="name" [style.background]="employee.color">
+      <h2>{{ employee.text }}</h2>
+    </div>
+    <div class="avatar">
+      <img [src]="employee.data.avatar" />
+    </div>
+    <div class="info" [style.color]="employee.color">
+      Age: {{ employee.data.age }}
+      <br />
+      <b>{{ employee.data.discipline }}</b>
```

Comment:
Let's get rid of ```<br>``` tag here:
```suggestion
      <div>Age: {{ employee.data.age }}</div>
      <div>
        <b>{{ employee.data.discipline }}</b>
      </div>
```